pkg/endpoint/endpoint.go

```diff
@@ -85,6 +85,12 @@ const (
 	// PropertySkipBPFRegeneration will mark the endpoint to skip ebpf
 	// regeneration.
 	PropertySkipBPFRegeneration = "property-skip-bpf-regeneration"
+
+	// PropertyCEPOwner will be able to store the CEP owner for this endpoint.
+	PropertyCEPOwner = "property-cep-owner"
+
+	// PropertyCEPName contains the CEP name for this endpoint.
+	PropertyCEPName = "property-cep-name-wrapper"
```

Comment:
Just a curiosity: why `property-cep-name-wrapper` and not `property-cep-name`?

Reply:
good shout, just a leftover I think :+1: